omrmakefiles/rules.zos.mk

```diff
@@ -139,7 +139,7 @@ endif
 	cp $< $*.c
 	xlc $(MCFLAGS) -I$(top_srcdir)/port/zos390/ -qmetal -qlongname -S -o $*.s $*.c > $*.asmlist
 	rm -f $*.c
-	as -mgoff -I CBC.SCCNSAM $*.s
+	as -mgoff $(UMA_MCASM_INCLUDES) -I CBC.SCCNSAM $*.s
```

Comment:
I don't expect "UMA" is appropriate here.

Reply:
So just `MCASM_INCLUDES`?

Reply:
Renamed to `MCASM_INCLUDES`.

Reply:
Fine with me.